docs/configure

```diff
@@ -672,20 +676,20 @@ virtual_domains:
                               { and {{<{$spam_score_int}{${extract{sa_tag}{$address_data}}} } \
                                      {eq {1}{${extract{on_spamassassin}{$address_data}}}} \
                                     } \
-                              } { eq {0}{${extract{on_spamassassin}{$address_data}}}} \
+                              } { eq {0}{${extract{dom_spamassassin}{$address_data}}}} \
```

Comment:
just translated to php, we would need this:
`if ((spamscore < limit && on_spamassassin==1) || on_spamassassin==0 || dom_spamassassin==0)`
or we could use your nice expression from above. I will have a look into it tomorrow. This is definitely one thing I should test as well.